source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/DateInputPropertiesTest.java

```diff
@@ -194,6 +199,103 @@ public DateInput castTo(BaseCardElement element) {
         }
     }
 
+    @Test
+    public void MinValidationTest() throws Exception
```

Comment:
Where are you expecting the exception to be thrown from ? Can we actually be specific about what specific Exception type will be thrown?